pkg/netconf/netconf_linux.go

```diff
@@ -208,6 +211,10 @@ func applyOuter(link netlink.Link, netCfg *NetworkConfig, wg *sync.WaitGroup, us
 	}
 
 	log.Debugf("Config(%s): %#v", linkName, match)
+	if match.WifiNetwork != "" {
+		match.DHCP = true
```

Comment:
I think we should add some comments here, it is necessary to explain why we set dhcp true.
```
We plan to use the dhcpcd hook to control the wpa_supplicant, https://wiki.archlinux.org/index.php/Dhcpcd#Hooks. Whether the Wi-Fi network uses DHCP or Static mode, the dhcpcd should be enabled by default, So we set `DHCP=true` here.
```